pystac_client/client.py

```diff
@@ -203,14 +209,14 @@ def from_file(  # type: ignore
 
         return client
 
+    def conforms_to(self, *conformance_classes: ConformanceClasses) -> bool:
+        return bool(self._stac_io and self._stac_io.conforms_to(*conformance_classes))
```

Comment:
Yeah I think conceptually it makes sense to have it on the Client. I think it would just take some refactoring of the other classes because they do not necessarily have access to the client class. I think they probably should though. I think that would make sense to specify `root` in all the classes and have that set to a `Client` obj.